modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -231,6 +231,20 @@ public ReturnType call(final String[] args, Cfg cfg) {
 
             // 5. options that can be influenced by the -d and -n arguments
 
+            if (options.getPort() != null) {
```

Comment:
I think this option can work with `-c, --config` as well. The `if (options.getPort() != null) {...}` statement can be moved above the `if (options.getConfig() != null) {...}` statement. This will require adding more test cases as well.